airsonic-main/src/main/java/org/airsonic/player/service/metadata/JaudiotaggerParser.java

```diff
@@ -131,13 +131,21 @@ public MetaData getRawMetaData(Path file) {
 
     private static String getTagField(Tag tag, FieldKey fieldKey) {
         try {
-            return StringUtils.trimToNull(tag.getFirst(fieldKey));
+            return stripNullCharacters(StringUtils.trimToNull(tag.getFirst(fieldKey)));
         } catch (Exception x) {
             // Ignored.
             return null;
         }
     }
 
+    private String stripNullCharacters(String s) {
+        if (s != null) {
+            return s.replace("\0", "");
+        } else {
+            return null;
+        }
```

Comment:
Can just use StringUtils.replace(s, "\0", " ");
It'll do all the null checks for you.

Reply:
When using StringUtils.replace() there is no need for a separate method:

```
--- a/airsonic-main/src/main/java/org/airsonic/player/service/metadata/JaudiotaggerParser.java
+++ b/airsonic-main/src/main/java/org/airsonic/player/service/metadata/JaudiotaggerParser.java
@@ -131,21 +131,13 @@ public class JaudiotaggerParser extends MetaDataParser {
 
     private static String getTagField(Tag tag, FieldKey fieldKey) {
         try {
-            return stripNullCharacters(StringUtils.trimToNull(tag.getFirst(fieldKey)));
+            return StringUtils.replace(StringUtils.trimToNull(tag.getFirst(fieldKey)), "\0", " ");
         } catch (Exception x) {
             // Ignored.
             return null;
         }
     }
 
-    private String stripNullCharacters(String s) {
-        if (s != null) {
-            return s.replace("\0", "");
-        } else {
-            return null;
-        }
-    }
-
     /**
      * Returns all tags supported by id3v1.
      */
```
This retains the original behaviour of returning null for empty tags by first using `trimToNull()`.

Reply:
Yup, that'll work, want to push a commit?